llvm/include/llvm/CodeGen/ValueTypes.h

```diff
@@ -87,6 +87,14 @@ namespace llvm {
       return getExtendedVectorVT(Context, VT, EC);
     }
 
+    /// Returns the EVT that represents a vector tuple type.
+    static EVT getRISCVVectorTupleVT(int Log2LMUL, unsigned NFields) {
+      // Sz = NF * LMUL * BitsPerBlock
+      unsigned Sz =
+          NFields * (Log2LMUL < 0 ? (64 >> -Log2LMUL) : (64 << Log2LMUL));
```

Comment:
Use RVVBitsPerBlock here. Maybe move the definition to the C++ file. We're going to need to change RVVBitsPerBlock based on ELEN and VLEN someday if we want to support VLEN==32 properly.

Though if we do that, we won't be able to get access to ELEN here.

Reply:
Maybe if we use `MinNumElts` rather than `RVVBitsPerBlock` approach we can keep it in the header file?

Reply:
Not the case if it's removed.